src/components/page-list.jsx

```diff
@@ -41,56 +42,28 @@ export default class PageList extends React.Component {
   renderHeader () {
     return (
       <tr>
-        <th>ID</th>
-        <th>Output Date</th>
-        <th>Site</th>
-        <th>Page Name</th>
-        <th>URL</th>
-        <th>Page View URL</th>
-        <th>Last Two</th>
-        <th>Latest to Base</th>
+        <th className="list__header--capture-date">Capture Date</th>
+        <th className="list__header--site">Site</th>
+        <th className="list__header--page-name">Page Name</th>
+        <th className="list__header--url">URL</th>
```

Comment:
Hmmm. There isn’t actually a `list` “block” that you’ve used here, so I think the “proper” BEM way to do this would be `container-list-view__header--[column name]`. You’re also supposed to keep the unmodified class, too:

```html
<th className="container-list-view__header container-list-view__header--url">URL</th>
```

It’s crazy verbose, but that is kind of the nature of BEM.

Another way to treat this might be something like:

```html
<th className="container-list-view__header" data-name="url">URL</th>
```

(You could also use `data-column="url"` or whatever other `data-something` you like.)

…and use it in CSS:

```css
.container-list-view__header[data-name="url"] {
  width: 40%;
}
```

But you could probably also keep it really simple just do:

```html
<th data-name="url">URL</th>
```

```css
.container-list-view th[data-name="url"] {
  width: 40%;
}
```

Usually I try and advocate *not* mixing BEM classes (e.g. `.container-list-view`) and regular element selectors (the `th`) this way (it’s easy to paint yourself into a corner when you realize you want to change a `<p>` into a `<div>` or something, for example), but I think it’s OK when the only acceptable elements are dictated to you by HTML’s vocabulary (i.e. because `.container-list-view` is always a table, you can’t really freely choose whether to use a `th` vs. some other element here. It’ll always be a `th`).

Anyway! What you’ve already got here is OK, too. But if you are trying to really do BEM here, I think it’s a little bit off.

Reply:
Thanks for the explanation! That was all very enlightening. There was one misunderstand though, `container-list-view` is not the class for the table. It's actually, what I see now, a modifier for the bootstrap class `container`. (So really should be `container--list-view`). I was imagining that `list` was the class for the table and it was a block contained in container 😵 

Realize now that we should be rid of all this container business once we drop bootstrap and go full flex/grid. I'm going to ignore it in this case. I changed `list` to the less generic `page-list`. Also, using modifiers for the headers "correctly" seems crazy verbose, so I went with the data attribute suggestion. I like this more lenient, sensical style.

How does it look now?

Reply:
Looks great! 👍